### PR TITLE
Changed the documentation to reflect the SDK API changes

### PR DESCRIPTION
**Pull request by juanjux**

### CODE REVIEW

File: driver/annotations.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Intro
 
-Once you have written the [AST parser](sdk.md) the next step is to write
+Once you have written the [native code to AST parser](sdk.md) the next step is to write
```
Comment (abeaumont) [inline driver/annotations.md]: _native parser_?

File: driver/internal-protocol.md [hunk 2]
```diff
@@ -12,12 +12,12 @@ It follows the same rules as the [driver protocol](protocol.html#transport).
 ## Encoding
 
 Inside the driver, internal communication between the UAST normalizer (Go) and
-the AST parser (native) uses JSON. Each message is encoded as a single-line
+the code to AST parser (native) uses JSON. Each message is encoded as a single-line
```
Comment (abeaumont) [inline driver/internal-protocol.md]: _native parser_?

File: driver/annotations.md (hunk 1 above)
Comment (juanjux) [inline driver/annotations.md]: I think this is clearer (it's the `intro.md` file), this is not the code API where we've to be succinct. Another option: *code to AST parser* (without the *native*).

File: driver/internal-protocol.md (hunk 2 above)
Comment (juanjux) [inline driver/internal-protocol.md]: See above.

File: driver/annotations.md (hunk 1 above)
Comment (dpaz) [inline driver/annotations.md]: code to AST sounds meter imo

Comment (abeaumont) [inline driver/annotations.md]: As you prefer, I just find *code to AST* redudant, while using _native parser_ is consistent.

File: user/server-grpc-example.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ func main() {
         os.Exit(1)
     }
     client := protocol.NewProtocolServiceClient(conn)
-    req := &protocol.ParseUASTRequest{Filename: "hello.py",
+    req := &protocol.ParseRequest{Filename: "hello.py",
```
Comment (dpaz) [inline user/server-grpc-example.md]: format these lines

File: uast/specification.md [hunk 4]
```diff
@@ -70,10 +70,10 @@ It is a 0-based index.
 **Col** is the column number in the original source code, relative to a line.
 It is a 1-based index.
 
-A position `(0, 0, 0)` means that position is not available. AST parsers
+A position `(0, 0, 0)` means that position is not available. native parsers
```
Comment (mcarmonaa) [inline uast/specification.md]: s/native/Native

File: user/server-grpc-example.md [hunk 5]
```diff
@@ -138,11 +138,11 @@ func main() {
 	}
 
 	client := protocol.NewProtocolServiceClient(conn)
-	req := &protocol.ParseUASTRequest{Filename: "hello.py",
+	req := &protocol.ParseRequest{Filename: "hello.py",
```
Comment (mcarmonaa) [inline user/server-grpc-example.md]: I think is more idiomatic write this as:

```go
req := &protocol.ParseRequest{
        Filename: "hello.py",
        Content: "print('hello world!')",
        Language: "python",
}
```

File: user/server-grpc-example.md [hunk 6]
```diff
@@ -45,46 +45,46 @@ Now we'll write a simple program that sends a request to get UAST of a simple
...
-                                      Content:  "print('hello world!')",
-                                      Language: "python",}
+	// Connect to the running server
+	conn, err := grpc.Dial("0.0.0.0:9432", grpc.WithTimeout(time.Second*2),
+		grpc.WithInsecure())
+	if err != nil {
+		os.Exit(1)
+	}
+	client := protocol.NewProtocolServiceClient(conn)
+	req := &protocol.ParseRequest{Filename: "hello.py",
```
Comment (mcarmonaa) [inline user/server-grpc-example.md]: I think is more idiomatic write this as:

```go
req := &protocol.ParseRequest{
        Filename: "hello.py",
        Content: "print('hello world!')",
        Language: "python",
}
```

**Comment by juanjux** [comment]
@mcarmonaa I've incorporated your feedback.

Travis build is failing, looks like some problem with the gitbook installation, I'm looking into it.

**Comment by juanjux** [comment]
For the record in case it happens to other node project: To fix the build I had to increase the node version for travis with a new `.nvmrc` file in the root of the repo with the latest LTS (6.11.1) version as content instead of the super old version that it uses by default.

**Pull request merged**